Including power in beaconData cause malformed URL since builder.setInclu...

alexdennis opened the pull request:
...deTxPowerLevel(false);

ryanbateman commented:
I believe my [previous pull request](https://github.com/google/uribeacon/pull/59) tried to handle the same issue but it was resolved as an old release that no longer reflects fixed code.

schilit commented:
I think the problem might be that the 16-bit UUID (FE D8) is included by Android by the addServiceData() so the first 2 bytes are duplicated. Try removing them and seeing how it works.

alexdennis commented:
I tested the lastest master however the malformed url still existed. So in my branch, I added back the flags and Adv Tx Power and removed the repeated UUID and its working for me.

g-ortuno commented on beacons/android/app/src/main/java/org/uribeacon/example/beacon/UriBeaconAdvertiserActivity.java:
```diff
@@ -55,16 +56,14 @@ private AdvertiseData getAdvertisementData() {
-        beaconData[8] = 0x08; // .org
+        byte[] beaconData = new byte[7];
+        beaconData[0] = 0x00; // flags
+        beaconData[1] = 0x20; // transmit power
```
Would you mind changing this to:
```java
beaconData[1] = (byte) 0xBA;
```

Which seems to be the correct Tx Power for the nexus 6.

alexdennis replied:
Certainly, doing now.

g-ortuno commented:
LGTM

schilit commented:
Thanks! Glad you persevered and got this working.
